mono/profiler/proflog.c

```diff
@@ -526,31 +541,36 @@ typedef struct {
 	uint64_t time;
 } MethodInfo;
 
-#ifdef TLS_INIT
-#undef TLS_INIT
-#endif
-
 #ifdef HOST_WIN32
```

Comment:
we should really kill this stuff and have the runtime provide tls for the profiler